appveyor.yml

```diff
@@ -12,6 +12,8 @@ branches:
   except:
     - gh-pages
 os: Windows Server 2012
+init:
+  - git config --global core.autocrlf true
```

Comment:
so also include this special instructions on README.md, cause otherwise, people that try to build this project w/o magic configuration will fail.

which still is a bad idea